packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-sell-button.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { getCardLabels, isValidToSell } from '@deriv/shared';
+import { Button, Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import { observer, useStore } from '@deriv/stores';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TAccumSellButton = {
+    contract_info?: TContractInfo;
+    current_stake: number | null;
+    currency?: string;
+    is_disabled: boolean;
+    is_sell_requested?: boolean;
+    onClick: (e: React.MouseEvent<HTMLElement>) => void;
+};
+const AccumulatorsSellButton = observer(
+    ({ contract_info, current_stake, currency, is_disabled, is_sell_requested, onClick }: TAccumSellButton) => {
+        const { ui } = useStore();
+        const { is_dark_mode_on } = ui;
+        const is_valid_to_sell = contract_info && isValidToSell(contract_info);
+        return (
+            <Fieldset className='trade-container__fieldset purchase-container__cell-button'>
+                <Button
+                    className='dc-btn--sell dc-btn__large'
+                    is_disabled={is_disabled || !is_valid_to_sell || is_sell_requested}
```

Comment:
Add extra disabling when Resale is not offered and when user click on 'Sell' btn (before showing 'Buy' btn) as it was requested